packages/adaptive_navigation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
```

Comment:
Please set this to release today with a breaking change version number, i.e. 0.1.0

Reply:
It's fine to update the version number as part of the PR making the change. I'll publish the package after the PR merges